spectacles/exceptions.py

```diff
@@ -120,7 +120,7 @@ def __init__(
         metadata = {
             "line_number": line_number,
             "lookml_url": lookml_url,
-            "dimension": field_name,
+            "field": field_name,
```

Comment:
Yeah, we would need to release this with a corresponding change in the app/UI. It will likely need to account for both dimension and field, given old runs will have the old key.